tests/test_freshness_date_parser.py

```diff
@@ -841,6 +871,7 @@ def test_relative_past_dates(self, date_string, ago, period):
         param('1 वर्ष, 8 महीने, 2 सप्ताह', ago={'years': 1, 'months': 8, 'weeks': 2}, period='week'),
         param('1 वर्ष 7 महीने', ago={'years': 1, 'months': 7}, period='month'),
         param('आज', ago={'days': 0}, period='day'),
+        param('1 दशक पहले', ago={'years': 10}, period='year'),
```

Comment:
Can we add a few more test cases for the Hindi version here? For e.g.

- 1 दशक पूर्व
- दो दशक पहले
- 10 दशकों पहले

Reply:
@gutsytechster Done this change, Please Look into it 😊.